plans/dht/test/common.go

```diff
@@ -420,7 +420,7 @@ func Connect(ctx context.Context, runenv *runtime.RunEnv, dht *kaddht.IpfsDHT, t
 		for i := 1; i <= attempts; i++ {
 			runenv.Message("dialling peer %s (attempt %d)", ai.ID, i)
 			select {
-			case <-time.After(time.Duration(rand.Intn(500)+100) * time.Millisecond):
+			case <-time.After(time.Duration(rand.Intn(500)+6000) * time.Millisecond):
```

Comment:
nit: `time.Duration(rand.Intn(500)) * time.Millisecond + 6 * time.Second`. Is 6s enough?

Reply:
the `dial backoff` in `libp2p-go-swarm` is set at `5*seconds` so it should be enough. I am not sure if it is exponential backoff, but when running this, I stopped seeing 5 attempts with `dial backoff error` and attempts were successful.

Now the testplan fails on a further stage.